lib/test/supplier_pade_test.dart

```diff
@@ -0,0 +1,26 @@
+import 'package:flutter/cupertino.dart';
+import 'package:flutter/material.dart';
+
+class SupplierPageTest extends StatelessWidget {
+  const SupplierPageTest({super.key});
```

Comment:
what is that? there should be no garbage in the project. Even if you test something, you must clean it. Cut it off.

Reply:
how i can delete it from pull request?